### ISSUE TITLE
Any news what next?

### ISSUE THREAD

**Issue by MESWEB** [renlok/WeBid#528]
@Nozlaf - any news what happened with @renlok?? Can we continue update this repo? What happened with website support?

**Comment by Nozlaf** [comment]
I have no idea I've changed companies and no-longer have an active interest in webid,

obviously as an open source project anyone can fork the code and improve on it as long as the original notices remain and attribution is provided to the project

I'd like to say that @renlok  provided excellent stewardship to this project over the past years, he was always very friendly and explained any issues he had with contributions that myself and my team provided to the project, I wish him the best for any future projects he would work on.

I also would like to thank yourself and anyone else who used the product, contributed to the code or tested / used the code, while many people probably never made any money with this code I certainly did save a significant amount of money on a project by using the code as the basis of our auction site.

**Comment by MESWEB** [comment]
What about you? Can we together update this great project to new standards? We can make new team with @denetwerkspecialist and make new functions and many more.

**Comment by Nozlaf** [comment]
I dont have the capacity for that sort of commitment, i'd suggest if @denetwerkspecialist forked the project and built on it properly in accordance with the licence that would be potentially the best place to contribute and @denetwerkspecialist would be the maintainer

**Comment by MESWEB** [comment]
I would prefer make more than one owner - next one can continue project if something would happened like in @renlok case. 
@Nozlaf - what about your knowledge with PHP and MYSQL ?? You helped me once upon a time and I was very satisfied from your help and experience - your code was great.
The best way is make a simple base of this project and after that we can make a mod for installing new functions like SMF project using. System read zip file including php code for add and next installing this code making backup if something gonna wrong. In future you can uninstall very easy old function and update it for new one. 
BTW I have new name for this project and we can together make $ from it by making similar one to ebay and make new auctions for free globally.
@denetwerkspecialist - what do you think about start new project all together?

**Comment by denetwerkspecialist** [comment]
We got some beta testers using the site to get more bugs and more user experience
Maybe next year will put back our update/bug fixed build on github.
But the payment system sucks (if you buy two or more items, you need to pay the items by items
No one invoice with multiple items
And the categorie problem is still an issue

**Comment by MESWEB** [comment]
> We got some beta testers using the site to get more bugs and more user experience
> Maybe next year will put back our update/bug fixed build on github.
> But the payment system sucks (if you buy two or more items, you need to pay the items by items
> No one invoice with multiple items
> And the categorie problem is still an issue

OK. Bro shout me when You will be ready for start update script or create new one. I can help. I have resolved issue PayPal integration.

**Comment by dutts** [comment]
Is it project dead then? I came across this looking for an open source auction script for a project and it looked promising, Sourceforge had v1.2.1 which I installed and setup, only to be told that v1.2.2 was the latest by the web admin page. Found v1.2.2 on github, copied that over my installation and ran the update.php script which appeared to report success but now it looks like some sort of localization issue has occurred and all of my strings on the site look like`{ L_363 }` codes rather than the actual prompts. I tried to sign up to the community support forums at http://www.webidsupport.com/forums/index.php but none of their OAuth2 connectors work and you can't sign up directly because the captcha on the signup page fails to load so now I'm stuck. Reticent to start a whole project based on a platform that appears to be dead with no support. Am I correct?

**Comment by MESWEB** [comment]
@dutts Yes project is dead. I'm only one who continue support little bit for this app. I just make my own upgrades.

**Comment by denetwerkspecialist** [comment]
We had it running and enhanced with many thing.
But we forked it wrong 
We may by put our version online when we get the proper fork rules done

**Comment by Nozlaf** [comment]
> We had it running and enhanced with many thing.
> But we forked it wrong
> We may by put our version online when we get the proper fork rules done

by forked it wrong you took out the copyright notices for the project and claimed it was yours, it's simple to fix but you chose to remove your fork instead and comment here several times about how you have this improved version

**Comment by MESWEB** [comment]
@denetwerkspecialist @Nozlaf Let's give him chance to improve this great app. But he need to remember about add copyright to @renlok. BTW he's still active on github and codding have a look here: https://github.com/renlok/renlok.github.io/commits/master

**Comment by dutts** [comment]
Ok so things not looking too great for the project as a whole, does anyone have any ideas why my upgrade from 1.2.1 to 1.2.2 should have caused those localization issues? 

It was literally a fresh install of 1.2.1 with zero modification once the install script had been ran and then immediately upgraded to 1.2.2.

**Comment by MESWEB** [comment]
@dutts just download this repo and install it or configure it for actually MySQL database.

**Comment by matbrgz** [comment]
how about start a fork refactoring entire code to typescript and react?? (up votes for interesting)

**Comment by MESWEB** [comment]
@MatheusRV  I'm in.
